core/src/main/java/org/lflang/generator/LFGenerator.java

```diff
@@ -61,6 +61,7 @@ public static FileConfig createFileConfig(
         case CPP -> new CppFileConfig(resource, srcGenBasePath, useHierarchicalBin);
         case Rust -> new RustFileConfig(resource, srcGenBasePath, useHierarchicalBin);
         case TS -> new TSFileConfig(resource, srcGenBasePath, useHierarchicalBin);
+        case UC -> throw new RuntimeException("Code-generation is not supported for the UC target yet");
```

Comment:
But it is supported! Perhaps a more informative error message? Point to the reactor-uc repo?